packages/rum/src/viewCollection.ts

```diff
@@ -195,13 +203,30 @@ function trackTimings(lifeCycle: LifeCycle, callback: (timings: Timings) => void
   return { stop: stopPerformanceTracking }
 }
 
-function trackLoadingTime(lifeCycle: LifeCycle, callback: (loadingTimeValue: number | undefined) => void) {
-  const startTime: number = performance.now()
+function trackLoadEventLoadingTime(
+  lifeCycle: LifeCycle,
+  startOrigin: number,
+  callback: (loadingTimeValue: number) => void
+) {
+  const startTime = performance.now()
+  const { unsubscribe: stopPerformanceTracking } = lifeCycle.subscribe(
+    LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED,
+    (entry) => {
+      if (entry.entryType === 'navigation') {
+        const navigationEntry = entry as PerformanceNavigationTiming
+        callback(msToNs(navigationEntry.loadEventEnd - startOrigin))
```

Comment:
```suggestion
        callback(msToNs(navigationEntry.loadEventEnd))
```
startOrigin should be 0 for initial view